TrelloAPI/DELETERequests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace TrelloAPI
+{
+    public class DELETERequests
+    {
+        private GETRequests _GETRequest;
+        public void DeleteBoard(string boardName)
+        {
+            _GETRequest = new GETRequests();
+            RestRequest deleteBoardRequest = new RestRequest($"/boards/{_GETRequest.GetSpecificBoard(boardName).id}?key={APIUserData.CorrectTestUser.Key}&token={APIUserData.CorrectTestUser.Token}", Method.DELETE);
```

Comment:
this line is too long, parameters like key and token could be passed using the query params